janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/JanusGraphConstants.java

```diff
@@ -73,6 +78,11 @@
             if (!v.isEmpty()) b.add(v);
         }
         COMPATIBLE_VERSIONS = b.build();
+        
+        for (String v : props.getProperty("titan.compatible-versions", "").split(",")) {
```

Comment:
I guess I like to pick up these changes as we go, especially seeing as the right side of the colon in the original for loop does a lot of work to get to something iterable. Your choice, im approving.